utilvolc/volcat_so2.py

```diff
@@ -40,7 +40,10 @@ def plotscatter(self,interp=False,cmap='viridis',neg=False):
         else:
             mass = pframe.massI.values
              
-        cb = plt.scatter(lon,lat,c=mass,s=0.1,cmap=cmap)
+        cb = plt.scatter(lon,lat,c=mass,s=0.1,cmap=cmap,
+        vmin=0.0, vmax=2.0)
+        plt.xlim(120, 195), plt.ylim(-35, -5)
```

Comment:
These are very specific to plotting a particular case and should just be temporary change - not committed.

Reply:
Modified.